http-server/src/main/java/io/micronaut/http/server/RouteExecutor.java

```diff
@@ -678,8 +687,7 @@ private CorePublisher<MutableHttpResponse<?>> fromReactiveExecute(PropagatedCont
             }
             return response.contextWrite(context -> ReactorPropagation.addPropagatedContext(context, propagatedContext).put(ServerRequestContext.KEY, request));
         }
-        MutableHttpResponse<?> response = forStatus(routeInfo, null).body(body);
-        return processPublisherBody(propagatedContext, request, response, routeInfo);
+        return processPublisherBody(propagatedContext, request, forStatus(routeInfo, null), false, publisher, routeInfo);
```

Comment:
why is the `.body` gone?

Reply:
I did a bit of refactoring to pass the converted publisher right away